src/DataExplorer/custom/bdbag.js

```diff
@@ -0,0 +1,94 @@
+import { fetchWithCreds } from '../../actions';
+import { constructGraphQLQuery, hasKeyChain, MSG_FAILED_DOWNLOAD } from '../utils.js';
+
+export const getBDBagQuery = referenceIDList => `{
```

Comment:
this is only supposed to work with DCP right?

Reply:
yes 😿